database/javascript/2017/16069.yaml

```diff
@@ -9,8 +9,6 @@ references:
 affected:
     - name: nodeffmpeg
       version:
-        - "==1.0.2"
```

Comment:
`<=1.0.2`

Reply:
https://www.npmjs.com/package/nodeffmpeg

According to NPM advisory it is not the latest version.

Reply:
"0.0.2-security" is the latest version according to NPM registry.

Reply:
I still feel it should be `<=1.0.2` . Even though other versions were unpublished, it is better to keep it that way

Reply:
According to this : https://registry.npmjs.org/nodeffmpeg
if we will use "<=1.0.2" then will miss out "0.0.2-security" tested locally.

Reply:
Then add `==0.0.2-security` and `==0.0.1-security` also as part of affected

Reply:
ohk!

Reply:
Done!